### PR TITLE
Adding documentation for docker usage

### PR DESCRIPTION
**Pull request by Stanislasss**
Waiting for review from @bersace

### CODE REVIEW

File: docs/cookbook.md [hunk 1]
```diff
@@ -229,3 +229,28 @@ The bug here, is that inspect does not truly inspect Postgres and always returns
...
 up to you to dig in `pg_catalog.pg_database.datacl` to find SQL GRANT.
+
+
+# ldap2pg as Docker container
+
+Already familiar with Docker and willing to save the setup time you're at the right place.
+
+To run the container simply use the command:
+``` console
+$ docker run dalibo/ldap2pg --help
```
Comment (bersace) [inline docs/cookbook.md]: I suggest `docker run --rm` for such example. It avoid polluting docker ps -a with one shot container.

File: docs/cookbook.md [hunk 2]
```diff
@@ -229,3 +229,28 @@ The bug here, is that inspect does not truly inspect Postgres and always returns
...
+```
+
+The Docker image of ldap2pg use the same configuration options as explained in the [cli](https://ldap2pg.readthedocs.io/en/latest/cli/) and [ldap2pg.yml](https://ldap2pg.readthedocs.io/en/latest/config/) sections.
```
Comment (bersace) [inline docs/cookbook.md]: You should point to internal links :

``` markdown
... [cli](cli.md) and [ldap2pg.yml](config.md) ...
```

File: docs/cookbook.md [hunk 3]
```diff
@@ -229,3 +229,28 @@ The bug here, is that inspect does not truly inspect Postgres and always returns
...
+You can either export the environmnet variables with the **-e** option:
+
+``` console
+$ docker run -e PGDSN=postgres://postgres@localhost:5432/ -e LDAPURI=ldaps://localhost -e LDAPBINDDN=cn=you,dc=entreprise,dc=fr -e LDAPPASSWORD=pasglop dalibo/ldap2pg
+```
+
+ or directly mount the ldap2pg.yml configuration file.
```
Comment (bersace) [inline docs/cookbook.md]: `ldap2pg.yml` is required for `sync_map`. Thus i would suggest to always mount `${PWD}/ldap2pg.yml`. The env var is an alternative to putting settings in `ldap2pg.yml` file, but `ldap2pg.yml` is still mandatory.

File: docs/cookbook.md (hunk 1 above)
Comment (Stanislasss) [reply docs/cookbook.md]: I didn't know the option, that's nice. Will be added.

File: docs/cookbook.md (hunk 2 above)
Comment (Stanislasss) [reply docs/cookbook.md]: Will change

File: docs/cookbook.md (hunk 3 above)
Comment (Stanislasss) [reply docs/cookbook.md]: It will be corrected.

**Pull request merged**